src/kernels/conv3x3.s

```diff
@@ -220,8 +201,7 @@ output_buffer_window = 4 * img_height * img_width
 .if limit_wave_cnt
   .SET_MAX_WAVES_LIMIT limit_wave_cnt
 .endif
-//xnack disabled by default
-//.SGPR_RESERVE_XNACK
+.SGPR_RESERVE_XNACK
```

Comment:
[Q] why this change?

Reply:
I asked and forgot. You have not answered. But this is the cause of the problem in amd-develop that led to the failed testing round in the staging. Let's be more attentive next time.

I will prepare hotfix.

Reply:
@Slimakanzer Please answer my initial question, thanks.